CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ Documentation for MIVisionX is available at
 ### Additions
 
 * Support for advanced GPUs
+* Support for PreEmphasis Filter augmentation in openVX extensions
```

Comment:
@swetha097 please sync to ToT. Pre-emphasis is not in this PR

Reply:
@swetha097 Please look at merge conflicts

Reply:
Merged with TOT @LakshmiKumar23